msg_broker/src/index.test.ts

```diff
@@ -3,13 +3,13 @@ const mock = { info: jest.fn() } as Record<keyof Logger, jest.Mock>
 jest.mock('../../utils/Logger/Logger', () => ({
   __esModule: true,
   Logger: class LoggerMock {
-    info = mock.info;
+    info = mock.info
   },
 }))
 
-
 import { main } from './index'
 
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
bof bof (english)

Reply:
It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring